src/EditorFeatures/CSharpTest/CommonMemberData.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests;
+
+public static class CommonMemberData
+{
+    public static IEnumerable<object[]> DataTypeDeclarationKeywords
+    {
+        get
+        {
+            yield return new[] { "class" };
+            yield return new[] { "struct" };
+            yield return new[] { "record" };
+            yield return new[] { "record class" };
+            yield return new[] { "record struct" };
+        }
+    }
+
+    public static IEnumerable<object[]> TypeDeclarationKeywords
+    {
+        get
+        {
+            return DataTypeDeclarationKeywords
+                .Concat([new object[] { "interface" }]);
+        }
+    }
```

Comment:
nit.  expression bodied members.

Reply:
nit: collection expressions.

Reply:
➡️ Removed this helper class since it was only being used in one location